src/Conflict.hs

```diff
@@ -45,6 +45,10 @@ setBodies f = runIdentity . bodies (Identity . f)
 setBodyStrings :: (String -> String) -> Conflict -> Conflict
 setBodyStrings f = runIdentity . bodyStrings (Identity . f)
 
+-- fold
+getBodies :: Conflict -> [[String]]
```

Comment:
Would be nicer to return a traversable of 3 a's?

Reply:
Done, changed `Conflict` to hold the data in `Sides` structures and applied this to other code using it as well.